pallets/rc-migrator/src/staking/delegated_staking.rs

```diff
@@ -157,16 +97,16 @@ impl<T: Config> PalletMigration for DelegatedStakingMigrator<T> {
 			last_key = match last_key {
 				DelegatedStakingStage::Delegators(last_key) => {
 					let mut delegators_iter = if let Some(last_key) = last_key.clone() {
-						alias::Delegations::<T>::iter_from(alias::Delegations::<T>::hashed_key_for(
-							last_key,
-						))
+						pallet_delegated_staking::Delegators::<T>::iter_from(
```

Comment:
why it used to be `Delegations` and now its `Delegators`?

Reply:
Yes this was the bug in the previous migration. The storage alias had the wrong name and did not read anything.